Library/Homebrew/cmd/command.rb

```diff
@@ -1,5 +1,6 @@
 module Homebrew
   def command
+    abort "This command requires a command argument" if ARGV.named.empty?
     cmd = ARGV.first
```

Comment:
I think it should be `abort "..." unless cmd` after this line. The major reason is there are internal commands start with `-`.